includes/Core/Modules/Modules.php

```diff
@@ -277,7 +275,7 @@ function( $token_response ) {
 				}
 
 				try {
-					$analytics = $this->get_module( Analytics::MODULE_SLUG );
+					$analytics = $this->get_module( Analytics_4::MODULE_SLUG );
 					$analytics->handle_token_response_data( $token_response );
```

Comment:
This method doesn't exist in `Analytics_4`. We need to investigate to find out if this method is still needed for `Analytics_4` and adapt it accordingly.